app/assets/stylesheets/home.scss

```diff
@@ -21,11 +20,25 @@
   }
 }
 
-div.module.text-center {
-  border-radius: 50px;
+.pie-size {
+  position: relative;
+  height: 300px;
+  background: asset-url("path.jpg") no-repeat center center;
 }
 
-.learn {
-  background-color: #d5ffea;
-  border-radius: 50px;
+.vertical-center {
+  position: absolute;
+  top: 25%;
+
+  input {
+    border-style: none;
+    background-color: rgba(0,0,0, .18);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.18)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.18` should be written with a leading zero as `0.18`<br>Commas in function arguments should be followed by a single space